src/rust/ide/src/controller/searcher.rs

```diff
@@ -438,20 +452,25 @@ mod test {
             let entry9 = searcher.database.get(9).unwrap();
             Fixture{searcher,entry1,entry2,entry9}
         }
+
+        fn new() -> Self {
+            Self::new_custom(|_,_| {})
+        }
     }
 
 
     #[wasm_bindgen_test]
     fn loading_list() {
         let mut test = TestWithLocalPoolExecutor::set_up();
-        let Fixture{searcher,entry1,entry9,..} = Fixture::new(|client| {
+        let Fixture{searcher,entry1,entry9,..} = Fixture::new_custom(|data,client| {
             let completion_response = language_server::response::Completion {
                 results: vec![1,5,9],
                 current_version: default(),
             };
+
             expect_call!(client.completion(
-                module      = "Test.Test".to_string(),
-                position    = TextLocation::at_document_begin().into(),
+                module      = data.module.path.file_path().clone(),
+                position    = TextLocation {line:0, column:5}.into(), // TODO should be something else likely
```

Comment:
Indeed, this should be another position: end of "main = 2+2" or whatever the body is.